vwr/driver.py

```diff
@@ -88,12 +88,28 @@ def turn_off(self):
         self.listener.settimeout(self.timeout)
         return (response == '!')
 
+    def check_fault(self):
+        """Checks for faults with the bath
+
+        Returns:
+            True if faulty, else False
+        """
+        self._send('RF')
+        response = self._receive()
+        return (response == '1')
+
     def get(self):
         """Gets the setpoint and internal temperature."""
-        response = {'setpoint': self.get_setpoint(),
-                    'actual': self.get_internal_temperature(),
-                    'pump': self.get_pump_speed(),
-                    'on': self.get_operating_status(),
+        setpoint = self.get_setpoint()
+        actual = self.get_internal_temperature()
+        pump = self.get_pump_speed()
+        on = self.get_operating_status()
+
+        response = {'setpoint': setpoint if setpoint else False,
+                    'actual': actual if actual else False,
+                    'pump': pump if pump else False,
+                    'on': on if on else False,
```

Comment:
First, the right notation for this would be `self.get_setpoint() or False`. Second, why change it to False? I think None is the right return type here.